### ISSUE TITLE
Appengine support

### ISSUE THREAD

**Issue by siriele** [stripe/stripe-go#56]
Although allowing the http client to be injected does indeed help with appengine usage line 68 of stripe.go is using an explicit Timeout in the http.Client struct literal. Appengine does not allow this. for now I'll be forking this but try using the Transport's deadline instead of the http.Client struct literal's timeout

**Comment by cosn** [comment]
Makes sense, thanks!

**Comment by cosn** [comment]
Sorry, but why is calling `stripe.SetHTTPClient(whatever_client_you_want)` sufficient?

**Comment by siriele** [comment]
Ahh I didn't see this. In appengine the struct for http.Client does not have the field timeout.

Further more when I just uses set client. The library worked in dev but when I deployed to prod I received a bunch of token invalid errors...then I had to switch to using the "client" so I could init the correct key. At this point deep inside the library it only cares for the http client it was initialized with..which is the default http client. Appengine just shuts this call down all together.
